templates/sidebar.html

```diff
@@ -110,9 +110,10 @@
             </div>
             
         </a>
-        <nav class="collapse col-md-4 col-lg-3 d-md-block bg-light sidebar" id="dash-sidebar">
+        <nav class="collapse col-md-4 col-lg-3 d-md-block bg-white sidebar" id="dash-sidebar">
```

Comment:
I just wanted the card's background to be white and not the complete background of sidebar.